apps/docs/src/components/home/CardsSection.tsx

```diff
@@ -17,6 +18,7 @@ export const CardsSection = () => {
       <div className="space-y-4">
         <FinancialPerformance />
         <StorageOverview />
+        <ActiveUsers />
```

Comment:
This looks a bit out of place, as it differs from all other containers. We do have something similar in the [App](https://lumada-design.github.io/uikit-app/master/)'s components page, but that page was kind of meant for debugging - this one is meant to be "pretty", right? 😅 

This looks good enough for me, but I'd like @francisco-guilherme's opinion on this too

![image](https://github.com/user-attachments/assets/4e846ae6-0d51-4969-919c-14dbe1378ef5)